website/docs/d/datasource_google_service_account_id_token.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_service_account_id_token"
+sidebar_current: "docs-google-service-account-id-token"
+description: |-
+  Produces OpenID Connect token for service accounts
+---
+
+# google\_service\_account\id\_token
+
+This data source provides a google OpenID Connect (`oidc`) `id_token`.  Tokens issued using this datasource are typically used to call external services that accept OIDC tokens for authentication (e.g. [Google Cloud Run](https://cloud.google.com/run/docs/authenticating/service-to-service)). 
+
+For more information see
+[OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
+
+## Example Usage - ServiceAccount JSON credential file.
+  Datasource will the service account provided by `GOOGLE_APPLICATION_CREDENTIALS` or `GOOGLE_CLOUD_KEYFILE_JSON` environment variable. 
+  
+  ```hcl
+  data "google_service_account_id_token" oidc {
+    target_audience = "https://foo.bar/"
+  }
+
+  output "oidc_token" {
+    value = data.google_service_account_id_token.oidc.id_token
+  }
+  ```
+
+## Example Usage - `Application Default Credentials`
```

Comment:
This is really the same as the previous example in the end- I made a suggestion for how to merge the two.

Reply:
must've missed the suggest; i guess you meant delete the dup so thats in here now